backend/composer.json

```diff
@@ -16,7 +16,8 @@
     },
     "require-dev": {
         "facade/ignition": "^2.5",
-        "laravel/sail": "^1.0.1",
+        "laravel-fans/docker": "^0.5.1",
+        "laravel/sail": "^1.13",
```

Comment:
composer remove laravel/sail

Reply:
https://github.com/framgia/sph_els_stephen/blob/8fb610f8f3fe5c71c480755444a656d39c9c4a2e/backend/composer.json

laravel sail is part of the laravel scaffolding sir